napari/layers/image/_tests/test_image.py

```diff
@@ -168,6 +170,7 @@ def test_rgba_image():
     assert layer._data_view.shape == shape[-3:]
 
 
+@pytest.mark.filterwarnings("ignore::UserWarning")
 def test_negative_rgba_image():
```

Comment:
what this test test if we cast to uint8?

Reply:
can't cast to uint8 as the `low` is out of range for uint8. For some reason in Vispy float rgb does work though, no matter the range so the warning is actually not correct.

Reply:
I adjusted the warning message and conditional now to reflect this.